Main/Source/char.cpp

```diff
@@ -5016,6 +5032,34 @@ void character::PrintEndVampirismMessage() const
     ADD_MESSAGE("You recall your delight of the morning sunshine back in New Attnam.");
 }
 
+void character::PrintBeginFearlessMessage () const
+{
+  if(IsPlayer())
+    ADD_MESSAGE("You feel very brave.");
```

Comment:
I think this is a little understated, considering the player is now feeling a little reckless (ignoring highly dangerous enemies). I'll try to come up with a suggestion. Maybe something about `blood coursing through your veins`

Reply:
Maybe we could go with understatement all the way? :D You feel a little bit more brave.

Or maybe You feel completely undefeatable!

Reply:
You shout: \\"I am invincible!\\"

(Monty python black knight joke)